app/pages/lab/landing-page.cjsx

```diff
@@ -0,0 +1,46 @@
+React = require 'react'
+counterpart = require 'counterpart'
+Translate = require 'react-translate-component'
+{Link} = require 'react-router'
+ZooniverseLogoType = require '../../partials/zooniverse-logotype'
+alert = require '../../lib/alert'
+LoginDialog = require '../../partials/login-dialog'
+
+counterpart.registerTranslations 'en',
+  landing:
+    title: "Zooniverse Project Builder"
+    content: "Anyone can build a Zooniverse project. Just upload your data and choose the tasks you want the volunteers to do. To find out more, read our How to Build a Project document, or click the button below to get started."
+    buttons:
+      getStarted: "Get started!"
+      signIn: "Sign in"
+      backToProjects: "Back to projects"
+
+module.exports = React.createClass
+  displayName: "ProjectBuilderLandingPage"
+
+  componentDidMount: ->
+    document.documentElement.classList.add 'on-secondary-page'
+    document.documentElement.classList.add 'on-landing-page'
+
+  componentWillUnmount: ->
+    document.documentElement.classList.remove 'on-secondary-page'
+    document.documentElement.classList.remove 'on-landing-page'
+
+  render: ->
+    <div className="landing-page">
+      <ZooniverseLogoType />
+      <h3 className="landing-title"><Translate content="landing.title" /></h3>
+      <p className="landing-tagline"><Translate content="landing.content" /></p>
+      {if @props.user?
+        <Link to="build" className="call-to-action standard-button landing-button" onClick={unless @props.user? then @showLoginDialog}><Translate content="landing.buttons.getStarted" /></Link>
```

Comment:
The `#/build` routes should actually be removed, everything is off `#/lab` now. I should have done that a while ago, sorry. I guess "Get started" should create a new project?